### PR TITLE
chore: update node and faucet urls for nova testnet

### PR DESCRIPTION
**Pull request by marc2332**
Closes https://github.com/iotaledger/firefly/issues/7868

### CODE REVIEW

File: packages/shared/lib/core/network/constants/explorer-urls.constant.ts [hunk 1]
```diff
@@ -4,5 +4,5 @@ export const EXPLORER_URLS: Readonly<{ [key in NetworkId]?: string }> = {
     [NetworkId.Iota]: 'https://explorer.iota.org/mainnet',
     [NetworkId.IotaAlphanet]: 'https://explorer.iota-alphanet.iotaledger.net/devnet',
     [NetworkId.Shimmer]: 'https://explorer.shimmer.network/shimmer',
-    [NetworkId.Testnet]: 'https://explorer.shimmer.network/testnet',
+    [NetworkId.Testnet]: '<UPDATE>',
```
Comment (begonaalvarezd) [inline packages/shared/lib/core/network/constants/explorer-urls.constant.ts:7]:
```suggestion
    [NetworkId.Testnet]: 'https://explorer.iota.works/iota2-testnet',
```

File: packages/shared/lib/core/network/constants/faucet-urls.constant.ts [hunk 2]
```diff
@@ -2,5 +2,5 @@ import { NetworkId } from '../enums'
 
 export const FAUCET_URLS: Readonly<{ [key in NetworkId]?: string }> = {
     [NetworkId.IotaAlphanet]: 'https://faucet.iota-alphanet.iotaledger.net/api/enqueue',
-    [NetworkId.Testnet]: 'https://faucet.testnet.shimmer.network/api/enqueue',
+    [NetworkId.Testnet]: 'https://faucet.nova-testnet.iotaledger.net/',
```
Comment (begonaalvarezd) [inline packages/shared/lib/core/network/constants/faucet-urls.constant.ts:5]:
```suggestion
    [NetworkId.Testnet]: 'https://faucet.nova-testnet.iotaledger.net/api/enqueue',
```

File: packages/shared/lib/core/network/tests/network.test.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ describe('File: network.ts', () => {
         [NetworkId.Iota]: ['https://api.stardust-mainnet.iotaledger.net', 'https://iota-node.tanglebay.com'],
         [NetworkId.IotaAlphanet]: ['https://api.iota-alphanet.iotaledger.net'],
         [NetworkId.Shimmer]: ['https://api.shimmer.network', 'https://shimmer-node.tanglebay.com'],
-        [NetworkId.Testnet]: ['https://api.testnet.shimmer.network'],
+        [NetworkId.Testnet]: ['https://api.nova-testnet.iotaledger.net/'],
```
Comment (begonaalvarezd) [inline packages/shared/lib/core/network/tests/network.test.ts:27]:
```suggestion
        [NetworkId.Testnet]: ['https://api.nova-testnet.iotaledger.net'],
```

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Comment by marc2332** [comment]
😭

**Comment by marc2332** [comment]
let me fix the test.

**Review by begonaalvarezd — APPROVED** [review]

**Review by cpl121 — APPROVED** [review]

**Pull request merged**